cli/provider/cmd.go

```diff
@@ -505,6 +508,25 @@ func (c *CmdConfigurator) ValidateFlags(ctx context.Context, cmd *cobra.Command)
 			utils.LogError(c.logger, err, errMsg)
 			return errors.New(errMsg)
 		}
+
+	case "templatize":
+		path := c.cfg.Path
```

Comment:
Converting the path into an absolute path and setting it as a config field is a repetitive code, It is used both in `templatize` and `normalize` command, please refactor this.